client/state/post-types/actions.js

```diff
@@ -39,8 +39,9 @@ export function requestPostTypes( siteId ) {
 			siteId
 		} );
 
-		return wpcom.withLocale().site( siteId ).postTypesList().then( ( { post_types: types } ) => {
-			dispatch( receivePostTypes( siteId, types ) );
+		return wpcom.undocumented().site( siteId ).postTypes()
+		.then( ( data ) => {
```

Comment:
Surprised that ESLint doesn't whine about this. I would normally expect a subsequent line to be tabbed if a continuation of the previous. Could we keep it as it was, especially since the newer variant is shorter than the previous already anyways?